src/lib/ui/sync-factory.js

```diff
@@ -26,6 +26,7 @@ module.exports = function(T, t, api) {
     // selfies
     currentpage: '1',
     events: {
+      'pane.activated': M('ui<T>Sync'),
```

Comment:
There is a sideeffect here, that when you go back to the TX pane it is always the DynamicContent loaded, I think you should store somewhere the last tab opened (probably localstorage) when the `ui<T>Tab` runs and use this information when this event is triggered to call the correct callback, e.g. `uiArticleSync`, `uiDynamicContentSync`, etc.. You can use Article as default.